packages/web-api/src/WebClient.ts

```diff
@@ -613,3 +614,21 @@ function warnDeprecations(method: string, logger: Logger): void {
     logger.warn(`${method} is deprecated. Please check on https://api.slack.com/methods for an alternative.`);
   }
 }
+
+/**
+ * Log a warning when using chat.postMessage without text argument
+ * @param method api method being called
+ * @param logger instance of we clients logger
+ */
+function warnMissingTextArgument(method: string, logger: Logger, options?: WebAPICallOptions): void {
+  const methodsWithOptionalText = ['chat.postMessage'];
```

Comment:
Can you add other chat.* methods here like I implemented in Python SDK? https://github.com/slackapi/python-slack-sdk/pull/910/files

Reply:
Thank you for revieing!

added warning logic for other methods and updated test :smile: